src/js/grapl-cdk/package-lock.json

```diff
@@ -3020,12 +3003,12 @@
                             }
                         },
                         "aws-sdk": {
-                            "version": "2.681.0",
-                            "resolved": "https://registry.yarnpkg.com/aws-sdk/-/aws-sdk-2.681.0.tgz#09eeedb5ca49813dfc637908abe408ae114a6824",
-                            "integrity": "sha512-/p8CDJ7LZvB1i4WrJrb32FUbbPdiZFZSN6FI2lv7s/scKypmuv/iJ9kpx6QWSWQZ72kJ3Njk/0o7GuVlw0jHXw==",
+                            "version": "2.722.0",
```

Comment:
this is a little scary, does CDK still work with this update?

Reply:
Tested a deployment and updated the original comment text.